packages/cli/src/cli/loaders/index.spec.ts

```diff
@@ -300,6 +300,194 @@ describe("bucket loaders", () => {
     });
   });
 
+  describe("ejs bucket loader", () => {
+    it("should handle static content in EJS templates", async () => {
+      setupFileMocks();
+
+      const input = `
+        <html>
+          <body>
+            <h1>Static Title</h1>
+            <p><%- user.name %></p>
+            <div>Static Text</div>
+          </body>
+        </html>
+      `.trim();
+
+      const expectedOutput = {
+        "body/0/0": "Static Title",
+        "body/1/0": "<%- user.name %>",
+        "body/2/0": "Static Text"
+      };
+
+      mockFileOperations(input);
+
+      const ejsLoader = createBucketLoader("ejs", "views/[locale]/template.ejs");
+      ejsLoader.setDefaultLocale("en");
+      const data = await ejsLoader.pull("en");
+
+      expect(data).toEqual(expectedOutput);
+    });
+
+    it("should handle EJS expressions in attributes", async () => {
+      setupFileMocks();
+
+      const input = `
+        <img alt="<%- imageAlt %>" src="/image.jpg">
+        <input placeholder="<%= placeholder %>">
+      `.trim();
+
+      const expectedOutput = {
+        "body/0#alt": "<%- imageAlt %>",
+        "body/1#placeholder": "<%= placeholder %>"
+      };
+
+      mockFileOperations(input);
+
+      const ejsLoader = createBucketLoader("ejs", "views/[locale]/template.ejs");
+      ejsLoader.setDefaultLocale("en");
+      const data = await ejsLoader.pull("en");
+
+      expect(data).toEqual(expectedOutput);
+    });
+
+    it("should preserve EJS logic blocks", async () => {
+      setupFileMocks();
+
+      const input = `
+        <ul>
+          <% users.forEach(function(user) { %>
+            <li><%- user.name %></li>
+          <% }); %>
+        </ul>
+      `.trim();
+
+      const expectedOutput = {
+        "body/0": "<% users.forEach(function(user) { %>",
+        "body/1/0": "<%- user.name %>",
+        "body/2": "<% }); %>"
+      };
+
+      mockFileOperations(input);
+
+      const ejsLoader = createBucketLoader("ejs", "views/[locale]/template.ejs");
+      ejsLoader.setDefaultLocale("en");
+      const data = await ejsLoader.pull("en");
+
+      expect(data).toEqual(expectedOutput);
+    });
+
+    it("should correctly push translations while preserving EJS syntax", async () => {
+      setupFileMocks();
+
+      const input = `
+        <h1>Welcome, <%- user.name %>!</h1>
+        <p>Your balance: <%= balance %></p>
+      `.trim();
+
+      const payload = {
+        "body/0/0": "Bienvenido, <%- user.name %>!",
+        "body/1/0": "Tu saldo: <%= balance %>"
+      };
+
+      const expectedOutput = `
+        <html lang="es">
+          <body>
+            <h1>Bienvenido, <%- user.name %>!</h1>
+            <p>Tu saldo: <%= balance %></p>
+          </body>
+        </html>
+      `.trim().replace(/\s+/g, ' ').replace(/>\s+</g, '><');
```

Comment:
I think this is slightly confusing, as it is not obvious what the expected output is. Can you include it as expected in the template literal here?

Reply:
I also noticed [the tests are failing](https://github.com/lingodotdev/lingo.dev/actions/runs/13307954002/job/37163185866?pr=453#step:12:140) - the strings seem correct, just whitespaces seem off.

Reply:
I agree with Matej here, 100%